contributing/publishing.md

```diff
@@ -91,98 +78,39 @@ Note that the beta branch, because of the unique versioning, should not be merge
-1. Select `Create Release Branch`.
-1. Approve the workflow in CircleCI:
-   1. Navigate to the `#pdt_releases` channel in Slack.
-   1. Soon you'll see a `Pending Approval for Creation of Release Branch` option. Click the `Visit Workflow` button to navigate to CircleCI.
-   1. Click the selection for `hold`.
-   1. Click the `Approve` button. See ![Approval View](../imgs/contributing-approval-button.png) for an example.
-1. Continue release normally, generating the change log.
+The steps used to execute a publish to the marketplace can be found in the assoicated github actions.
 
 ## Generating a Major Release
 
 The versioning we follow is intentionally mapped with Salesforce Core. When a major version bump occurs, such as 53.0 -> 54.0, we release a major version update as well.
 
-1. Open the Command Palette (press Ctrl+Shift+P on Windows or Linux, or Cmd+Shift+P on macOS).
-1. Search for `Tasks: Run Task`.
-1. Select `Create Release Branch`.
-1. Select 'Major' as the version.
-1. Continue the rest of the release branch process documented above.
-1. When generating the change log, manually input the major release version (ex: 54.0.0)
-1. Do not publish the minor version generated in CircleCI.
+## Downloading the .vsix from Github Action
 
-## Downloading the .vsix from CircleCI
+### Options
 
-### Prerequisite
-
-- Lerna is properly installed (`npm install -g lerna@3.13.1`).
-- You've created a CircleCI token that grants you access to the artifacts generated per build. More info on CircleCI's doc [Create a Personal API token](https://circleci.com/docs/2.0/managing-api-tokens/#creating-a-personal-api-token).
-- All tests have been run prior to publishing. We don't run the tests during the
-  publishing cycle since it generates artifacts that we do not want to include
-  in the packaged extensions.
-
-### Steps
-
-1. `npm install` to install all the dependencies and to symlink interdependent
-   local modules.
-1. You have set the `SALESFORCEDX_VSCODE_VERSION`, `CIRCLECI_TOKEN` and
-   `CIRCLECI_BUILD` environment variables that give you access to download the
-   CircleCI artifacts.
-1. `npm run circleci:artifacts` downloads _each_ extension artifact as a .vsix
-   and stores it in the corresponding packages/salesforcedx-vscode-\* path.
+- Download directly from the Github Action run. You will find artifacts that are assoicated with a run at the bottom of the summary screen
+- Use the gh cli to download artifacts. `gh run download --dir /dir/where/you/want/the/vsix/files/ 3746978326`. The last arg is the GHA job id. This can be found in the UI or by executing `gh run list`.
 
 **At this stage, it is possible to share the .vsix directly for manual
 installation.**
 
+To manually install vsix files you can use the `code` or `code-insiders` cli.
+
+- `code-insiders --install-extension /path/to/the/vsix/iama.vsix`
+- or install all downloaded vsix files `find ./vsix/download/path -type f -name "*.vsix" -exec code --install-extension {} \;`
+
```

Comment:
I outlined how you can download the vsix artifacts from GHA here and how to easily install those.  let me know if you think we need an in between step for something closer to ctv, but it seems pretty straightforward.